src/Token/_tokenButtonUtils.ts

```diff
@@ -0,0 +1,61 @@
+import {css} from 'styled-components'
+import {variant} from 'styled-system'
+import {get} from '../constants'
+import {tokenSizes, TokenSizeKeys, defaultTokenSize} from './TokenBase'
+
+export interface TokenButtonProps {
+  borderOffset?: number
+  parentTokenTag: 'span' | 'button' | 'a'
+  variant?: TokenSizeKeys
+  parentTokenIsInteractive?: boolean
+}
+
+export const variants = variant({
+  variants: {
+    sm: {
+      height: `${tokenSizes.sm}px`,
+      width: `${tokenSizes.sm}px`
+    },
+    md: {
+      height: `${tokenSizes.md}px`,
+      width: `${tokenSizes.md}px`
+    },
+    lg: {
+      height: `${tokenSizes.lg}px`,
+      width: `${tokenSizes.lg}px`
+    },
+    xl: {
+      height: `${tokenSizes.xl}px`,
+      width: `${tokenSizes.xl}px`
+    }
+  }
+})
+
+export const tokenButtonStyles = css`
```

Comment:
Is there a reason you pulled this into its own file?

Reply:
I had an "add" button at some point, and I just forgot to move this back to the `RemoveTokenButton` component file.